### PR TITLE
add selecting a default cluster

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/web-modeler/issues/9958.

Details how to select a default cluster for process applications.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/process-applications.md [hunk 1]
```diff
@@ -24,10 +24,22 @@ main process depends on, such as called supporting processes, DMN decisions, or
...
+
+After selecting **Create process application**, you may also select a preferred development cluster when creating the process application.
+
+If you have already [created a cluster](/guides/create-cluster.md), this will display below the **Name** for your process application, under **Select a development cluster**.
+
+Select the toggle next to the preferred development cluster. The selected cluster will be used as the preferred cluster for deployment during development.
+
+If you have not yet created a cluster, or see **No Cluster has been created yet** in the modal, select **Create new dev cluster**. This will automatically take you to the **Clusters** tab in Console to create a new cluster.
+
+If your modeling plan does not permit you to create more clusters, you may notice a warning message reading `Cluster creation is not available on this modeling plan`. Contact the admin or owner of your organization to upgrade your plan.
```
Comment (vpellegrino) [inline docs/components/modeler/web-modeler/process-applications.md:41]: Just FYI, (up to you to include it or not in the docs), we also support the case that the user has no permission to create a cluster. In this case, you will see a warning message:
`Missing permissions`. Contact your organization admin to create a DEV cluster.

**Review by vpellegrino — APPROVED** [review]
✅ Nice doc addition, @christinaausley!
I just left one addition (it's up to you to decide whether to include it or not).

File: docs/components/modeler/web-modeler/process-applications.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/modeler/web-modeler/process-applications.md:41]: Added 😄

File: docs/components/modeler/web-modeler/process-applications.md [hunk 2]
```diff
@@ -24,10 +24,24 @@ main process depends on, such as called supporting processes, DMN decisions, or
...
+
+If you notice a warning message reading `Missing permissions`, you may not have the proper permissions to create a cluster. Contact your organization's admin to create a cluster.
```
Comment (conceptualshark) [inline docs/components/modeler/web-modeler/process-applications.md:43]:
```suggestion
:::note 
Both your plan and account must have sufficient permissions to create new clusters. If you experience `Cluster creation is not available on this modeling plan` or `Missing permissions` warnings, contact the admin or owner of your organization to create a cluster, or to upgrade your plan.
:::
```

Comment (christinaausley) [reply docs/components/modeler/web-modeler/process-applications.md:43]: I do want to differentiate the two solutions based on the error messages. I'm not going to commit this, but I will tweak it a bit for conciseness!

File: docs/components/modeler/web-modeler/process-applications.md [hunk 3]
```diff
@@ -24,10 +24,22 @@ main process depends on, such as called supporting processes, DMN decisions, or
...
 
 Create a new process application by clicking the **Create process application** button in an empty project or via
-**Create new > Process application** on the project or folder page.
+**New project > Create process application** if you have not yet created a project.
 
 <p><img src={EmptyProjectImg} alt="Create a process application from an empty project" /></p>
 
+### Select a default cluster
+
+After selecting **Create process application**, you may also select a preferred development cluster when creating the process application.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/modeler/web-modeler/process-applications.md]: There is a lot of context alluding to this prior to this section in the doc. For example, "We advise using a process application for all non-trivial automation projects." I don't want to be redundant, but I will add some clarity in my next commit 👍

**Comment by christinaausley** [comment]
> @christinaausley, one further addition is that this feature is SaaS only, and we need to make sure that the documentation reflects this specificity.

I've added a badge for clarity 👍

**Review by vpellegrino — APPROVED** [review]
Awesome! 🙌🏼

**Pull request merged**